contributing/code/pull_requests.rst

```diff
@@ -400,6 +400,33 @@ The :doc:`core team </contributing/code/core_team>` is responsible for deciding
 which PR gets merged, so their feedback is the most relevant. So do not feel
 pressured to refactor your code immediately when someone provides feedback.
 
+Automated Feedback
+~~~~~~~~~~~~~~~~~~
+
+There are many automated scripts that will provide feedback on your PR.
+
+fabbot
+""""""
+
+`fabbot`_ will review code style, check for common typos and make sure your git
+history looks good. If there are any issues, fabbot will often suggest what changes
+that should be done. Most of the time you get a command to run to automatically
+fix the changes.
+
+Psalm
+"""""
+
+`Psalm`_ will make a comment on your Pull Request if it discovers any potential
+type errors. The Psalm errors are not always correct, but each should be reviewed
+and discussed. A Pull Request should not update the psalm baseline nor add ``@psalm-``
+annotations.
+
+Psalm can run locally with:
+
+.. code-block:: terminal
+
+    $ psalm src/Symfony/Component/Workflow
```

Comment:
this requires `psalm` to be installed, which is not the case right?

Reply:
Yes. But the official [installation](https://psalm.dev/docs/running_psalm/installation/) says that you should install psalm with the local composer... 

What I would wish for is to link to psalm docs that says: "Here is how you install the phar with `wget`" or "here is how you install psalm with global composer".

Reply:
Or we require it in `require-dev` of Symfony? (which is the official way to use Psalm, if we don't want mix-ups we can also require `psalm/phar@stable` like you do in the CI pipeline already)

Reply:
This is my highly personal opinion, but I think the official way to install psalm is terrible. 😄  Psalm, PHPUnit, PHPStan are tools to help you code and develop. Just like PHPStorm. They are not a dependency of your project. You should also not "suffer" if such dependency is conflicting with your project dependencies. 

Adding `psalm/phar` is just a "quick fix" to the problem. The real solution is to like PHPUnit suggest and `wget` the phar. 

(PHPUnit is special because it is both a dependency (`PHPUnit\Framework\TestCase` and other classes) and a script that execute some code)

To further show my logic, If Psalm was written in GoLang or Ruby, would you ever think "I wish I could download this with composer".

---------

I will discuss this at psalm and make sure they update their docs.

Reply:
I've added a link to the psalm installation instructions. 

I hope psalm will merge my PR: https://github.com/vimeo/psalm/pull/5274

Reply:
The psalm installation instructions are updated